### PR TITLE
31432: GlueDB - Custom 'renewal plan_change' EDI workflow

### PR DESCRIPTION
**Pull request by nks2109**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [31432](https://devops.dchbx.org/redmine/issues/31432) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | n/a |
| Peer Review Story Points | 3 |
| Ticket Author | @kevin-lerer |

### CODE REVIEW

File: spec/models/enrollment_action/simple_renewal_spec.rb [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    allow(action_publish_helper).to receive(:keep_member_ends).with([])
+  end
+
+  it "publishes termination resulting xml to edi" do
+    expect(subject).to receive(:publish_edi).with(amqp_connection, termination_helper_result_xml, terminated_policy_eg_id, employer_hbx_id)
+    subject.publish
+  end
+
+  it "publishes an event of enrollment auto renewing" do
+    expect(action_publish_helper).to receive(:set_event_action).with("urn:openhbx:terms:v1:enrollment#auto_renew")
```
Comment (TreyE) [inline spec/models/enrollment_action/simple_renewal_spec.rb]: This is not the correct event type.  It should be "initial".

File: spec/models/enrollment_action/simple_renewal_spec.rb [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    subject.publish
+  end
+
+  it "publishes auto renewing resulting xml to edi" do
+    expect(subject).to receive(:publish_edi).with(amqp_connection, action_helper_result_xml, action_event.hbx_enrollment_id, action_event.employer_hbx_id)
+    subject.publish
+  end
+
+  it "publishes an event of type auto renew" do
+    expect(action_publish_helper).to receive(:set_event_action).
```
Comment (TreyE) [inline spec/models/enrollment_action/simple_renewal_spec.rb]: See above on the setting of the event.

**Review by TreyE — CHANGES_REQUESTED** [review]
Please see notes.

File: spec/models/enrollment_action/simple_renewal_spec.rb (hunk 1 above)
Comment (nks2109) [reply spec/models/enrollment_action/simple_renewal_spec.rb]: please do review it changed event type from auto_renew to active_renew

**Review by TreyE — APPROVED** [review]
Approved with new event.

**Pull request merged**